public/app/themes/justice/inc/admin.php

```diff
@@ -36,4 +37,41 @@ public static function removeCustomizer()
         $customizer_url = add_query_arg('return', urlencode(remove_query_arg(wp_removable_query_args(), wp_unslash($_SERVER['REQUEST_URI']))), 'customize.php');
         remove_submenu_page('themes.php', $customizer_url);
     }
+
+    public static function removePublishPressUpsellBranding()
+    {
+        echo '<style>
+               .toplevel_page_revisionary-q {
+                 li:last-of-type {
+                   display: none;
```

Comment:
This is too broad sweeping - it targets every list on the Revision Queue page. e.g. There's a 'Collapse menu' button at the bottom of the sidebar, this css rule set's that to display none.

This rule doesn't target other pages, i.e. Revisions Archive or Settings page of this plugin.

Perhaps this link could be targeted by an attribute?

Reply:
Thanks, I've changed it to target the link with the matching href.